### PR TITLE
Send email on success

### PR DESCRIPTION
**Pull request by mjenczmyk**
There are two commits in this pull request - the first one cleans up the script body to make it more clear (but shouldn't change behaviour of the script), the second one adds sending message on book grabbing success and improves message being send on script failure.

Please, see if you're OK with this changes. It may be easier to review if you verified those changes separately. Let me know if you would like to split in into two pull requests.

### CODE REVIEW

File: src/packtPublishingFreeEbook.py [hunk 1]
```diff
@@ -118,6 +126,7 @@ def __init__(self, currentSession):
         self.session = currentSession.getCurrentHttpSession()
         self.accountData = currentSession.getCurrentConfig()
         self.bookTitle = ""
+        self.pretty_book_title = None
```
Comment (mjenczmyk) [inline src/packtPublishingFreeEbook.py]: I've added this new field that holds human friendly book title.

**Comment by igbt6** [comment]
hello, im back, will take a look on your changes tommorow ;>

**Comment by igbt6** [comment]
merged manually at: 5336991b9ce0f1ef6eeb96450c41e0c18b18bfa2 , thx!

**Comment by mjenczmyk** [comment]
When merging manually you forgot about updating `README.md` (which should be updated as code behavior has changed), you also didn't merge addition of `BookGraber.pretty_book_title` - is there because you didn't want to apply these changes or is there another reason?

Let's talk about changes, applying them selectively and manually without discussion makes me feel that my work ends at proposing the changes and that I have nothing to say afterwards.

File: src/packtPublishingFreeEbook.py (hunk 1 above)
Comment (igbt6) [inline src/packtPublishingFreeEbook.py]: I think, there is no need to create another 'title' variable which will be only used for sending status email

**Comment by igbt6** [comment]
hey, you can always add your comments here, and we can reopen the request if sth more to add. I updated comments, README has been updated in the next commit.

**Pull request closed without merging**